### DISCUSSION TITLE
osu! lazer import issues

### DISCUSSION THREAD

**Discussion by shroomerh** [ppy/osu#27946] [Q&A]
### Type

Game behaviour

### Bug description

Went to go play lazer to join a friend and as I tried to download a map I got stuck on "Beatmap import is initializing" .
I started out by switching my download directory which seemed like an obvious solution, which did not work, so I went to setup wizard, then I reinstalled the game and I've had no luck. I've searched the internet and I can't really seem to find this issue anywhere else.
It also doesn't seem to be using either of my ssds at all.
![image](https://github.com/ppy/osu/assets/167600072/406387fd-2572-42df-b2fd-39dcf099e052)

### Screenshots or videos

![osu_2024-04-19_21-56-39](https://github.com/ppy/osu/assets/167600072/ebd74a5f-c0a4-4e45-99cb-63bfd0f3ef91)
![osu_2024-04-19_21-56-35](https://github.com/ppy/osu/assets/167600072/b6bc67ed-e95a-4ebe-86db-a9e79c49eb84)
![osu_2024-04-19_21-55-27](https://github.com/ppy/osu/assets/167600072/36567247-2b15-446c-92de-6ea0c4c59063)

### Version

2024.412.1

### Logs

[compressed-logs.zip](https://github.com/ppy/osu/files/15046910/compressed-logs.zip)

**Comment by peppy** [comment]
Judging by logs you might be trying to import a very broken beatmap.

**Reply by murryguy** [reply]
Then what is he gonna do?

**Comment by shroomerh** [comment]
Oh okay, I have one in mind, thank you for your quick response.
